packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -225,6 +226,18 @@ mixin SchedulerBinding on BindingBase, ServicesBinding {
   AppLifecycleState get lifecycleState => _lifecycleState;
   AppLifecycleState _lifecycleState;
 
+  /// Initializes the [lifecycleState] with the data from the window.
+  ///
+  /// An optional [testWindow] may be provided to supply test values.
+  ///
+  /// Once the lifecycleState is initialized, it cannot be initialized again.
+  @protected
+  void initLifecycleState([Window testWindow]) {
```

Comment:
This method can internally use the `window` accessor of the binding instead of taking in the property.  Is there a situation where a different `Window` instance would be appropriate?

Reply:
I must have been doing something wrong somewhere else, which led me to do this, but then accidentally fixed it at some point because the window accessor is working fine (but was not at some point)